### DISCUSSION TITLE
A little Shoutout for the project

### DISCUSSION THREAD

**Discussion by TheInternetHelpdesk** [LibreHardwareMonitor/LibreHardwareMonitor#1390] [General]
This project was just featured on JaysTwoCents. He had some good ideas on ways to alter and improve a few bits, mostly cosmetic.

https://youtu.be/sX2ThUWNXio

**Comment by SnabbSpelare** [comment]
### Awesome program!
**I come from Jayz aswell!** _He had some suggestions hopefully they'll be considered_!

**Comment by STRETCHDEATH** [comment]
here from jay aswell, moved from hwmonitor to libre, currently running latest released action and its running smooth as! loving it and keen for 0.9.4 :P keep up the good work devs and contributers!!!!!!
